Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/types.go

```diff
@@ -2523,6 +2526,12 @@ type SecurityContextConstraints struct {
 	SupplementalGroups SupplementalGroupsStrategyOptions
 	// FSGroup is the strategy that will dictate what fs group is used by the SecurityContext.
 	FSGroup FSGroupStrategyOptions
+	// ReadOnlyRootFilesystem when set to true will force containers to run with a read only root file
+	// system.  If the container specifically requests to run with a non-read only root file system
+	// the SCC should deny the pod.
+	// If set to false the container may run with a read only root file system if it wishes but it
+	// will not be forced to.
+	ReadOnlyRootFilesystem bool
```

Comment:
I can't think of a use case for prevention.  Unless it was a bug in Docker or something that allowed root compromise, and we'd probably still have to fix that right away.